atlasdb-config/src/test/java/com/palantir/atlasdb/http/AtlasDbErrorDecoderTest.java

```diff
@@ -87,16 +114,17 @@ public void shouldDelegateToDefaultDecoderWhen503AndRetryableException() {
         assertThat(exception, is(sameInstance(RETRYABLE_EXCEPTION)));
     }
 
+    // Not 503 nor 429 tests
     @Test
-    public void shouldDelegateToDefaultDecoderWhenNeither503NorRetryableException() {
-        Response response = makeDefaultDecoderReplyWhenReceivingResponse(STATUS_NOT_503, NON_RETRYABLE_EXCEPTION);
+    public void shouldDelegateToDefaultDecoderWhenNeither503Nor429NorRetryableException() {
+        Response response = makeDefaultDecoderReplyWhenReceivingResponse(STATUS_NOT_503_NOR_429, NON_RETRYABLE_EXCEPTION);
         Exception exception = atlasDbDecoder.decode(EMPTY_METHOD_KEY, response);
         assertThat(exception, is(sameInstance(NON_RETRYABLE_EXCEPTION)));
     }
 
     @Test
-    public void shouldDelegateToDefaultDecoderWhenNot503AndRetryableException() {
-        Response response = makeDefaultDecoderReplyWhenReceivingResponse(STATUS_NOT_503, RETRYABLE_EXCEPTION);
+    public void shouldDelegateToDefaultDecoderWhenNot503Nor429AndRetryableException() {
```

Comment:
nit: for consistency with the above test name I'd prefer `...WhenNeither503Nor429AndRetryableException`